intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationship.java

```diff
@@ -85,11 +88,13 @@ public class AtlasRelationship extends AtlasStruct implements Serializable {
     private Date          updateTime     = null;
     private Long          version        = 0L;
 
-    public enum Status { ACTIVE, DELETED }
+    public enum Status { ACTIVE, DELETED, PURGE_DUE_TO_ENTITY }
 
     private Set<AtlasClassification> propagatedClassifications;
     private Set<AtlasClassification> blockedPropagatedClassifications;
 
+    private Map<String, Object> customRelationshipInfo;
```

Comment:
I think the variable `customRelationshipInfo` isnt used.

Reply:
It is used in the Kafka message. Check setCustomRelationshipInfo(customRelationshipInfo) in AtlasRelationshipHeader method.